docs/components/movement-sensor/gps/gps-rtk.md

```diff
@@ -88,13 +89,13 @@ Click **Create Component** and then fill in the attributes for your model.
         "correction_source": "ntrip",
         "serial_attributes": {
           "serial_baud_rate": 115200,
-          "serial_path": "/dev/serial/by-path/<device_ID>"
+          "serial_path": "/dev/serial/by-path/12335"

```

Comment:
```suggestion
          "serial_path": "/dev/serial/by-path/12345"
```
[opt] not sure if this is what was intended to make it more obviously generic

Reply:
Nice, that does help!